### PR TITLE
Feature/card article tag

### PR DESCRIPTION
**Pull request by JoanClaret**
This PR includes 2 main features:

- Modify tag-chip so it can receive a `className`
- Include `sui-tag-chip` in card-article instead of using a link.

### CODE REVIEW

File: components/card/article/src/index.scss [hunk 1]
```diff
@@ -28,17 +28,12 @@ $c-article-tag-types: () !default;
 
   &-tag {
     @include reset-link;
-    background-color: $c-article-tag;
-    border-radius: $bdrs-small;
-    color: $c-white;
-    font-size: $fz-s;
-    font-weight: $fw-bold;
-    padding: $p-v-xsmall $p-h-xsmall;
-    text-transform: uppercase;
```
Comment (kikoruiz) [inline components/card/article/src/index.scss]: Missing uppercase.

File: components/tag/chip/src/index.js [hunk 2]
```diff
@@ -6,8 +6,9 @@ const Tag = ({Link, children, url, ...rest} = {}) => url
   ? <Link href={url} {...rest}>{children}</Link>
   : <span {...rest}>{children}</span>
 
-const tagChipClassName = ({isClickable}) => cx('sui-TagChip', {
-  'sui-TagChip-link': isClickable
+const tagChipClassName = ({isClickable, className}) => cx('sui-TagChip', {
+  'sui-TagChip-link': isClickable,
+  [className]: className
```
Comment (kikoruiz) [inline components/tag/chip/src/index.js]: I think it's not needed. In any case, you can define a default prop as empty string.

Comment (JoanClaret) [reply components/tag/chip/src/index.js]: Fixed!

File: components/card/article/src/index.scss (hunk 1 above)
Comment (JoanClaret) [reply components/card/article/src/index.scss]: yeah! done.

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**